roles/wildfly_firewalld/tasks/firewalld.yml

```diff
@@ -6,13 +6,13 @@
     quiet: true
 
 - name: "Ensure firewalld is available."
-  become: true
+  become: "{{ root_permission | default(true) }}"
   ansible.builtin.package:
     name: "{{ wildfly_firewalld_package_name }}"
     state: present
 
 - name: "Ensure firewalld is running."
-  become: true
+  become: "{{ root_permission | default(true) }}"
```

Comment:
Ditto

Reply:
There is little a collection can do without become set to true.